conformity/resource_conformity_custom_rule.go

```diff
@@ -370,6 +370,8 @@ func processInputCustomRuleConditions(conditionsIn []interface{}) []cloudconform
 			obj.Value, _ = strconv.ParseBool(m["value"].(string))
 		} else if numValue, err := strconv.Atoi(m["value"].(string)); err == nil {
 			obj.Value = numValue
+		} else if m["value"].(string) == "null" {
+			obj.Value = nil
```

Comment:
⛏️ Comment will help future reader.
```suggestion
			// JSON Marshal will convert to null before being passed to custom rules API
			obj.Value = nil
```